.meta.toml

```diff
@@ -0,0 +1,29 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/zope-product
+[meta]
+template = "zope-product"
+commit-id = "c9ccfc6dd58798357db94e3aaf213ee82a4e8400"
+
+[python]
+with-appveyor = false
+with-pypy = false
+with-legacy-python = true
+with-docs = false
+with-sphinx-doctests = false
+
+[tox]
+use-flake8 = true
+
+[coverage]
+fail-under = 65
```

Comment:
Previously we had 75 % coverage (see setup.cfg) – was this simply wrong or did the coverage actually drop?
GHA counts 66 % (see https://github.com/zopefoundation/Products.ZSQLMethods/runs/2107041611?check_suite_focus=true)

Reply:
The reason is simple: On `master` the test runner only visits `src/Products/ZSQLMethods` and doesn't visit the path `src/DC/Shared/ZRDB`. The old configuration was broken.